src/config_file.rs

```diff
@@ -1,6 +1,7 @@
 use crate::flags::display::Display;
 use crate::flags::icons::{IconOption, IconTheme};
 use crate::flags::layout::Layout;
+use crate::flags::permissions::PermissionFlag;
```

Comment:
how about also change this to singular?

Reply:
I was kinda on the fence on that one. On one had, using `permissions` will keep it consistent with `src/meta/permissions.rs`, but on the other hand everything inside it is `permission`.
Anyways, I don't really have a strong opinion on it. I'll change it to permission.